app/src/main/res/layout/fragment_filter.xml

```diff
@@ -0,0 +1,44 @@
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/filter_bottom_sheet_heading"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/filter_frag_heading_height"
+        android:gravity="center"
+        android:text="@string/filter_bottom_sheet_title"
+        android:textSize="@dimen/filter_frag_heading_size" />
```

Comment:
I don't think this needs this since there are no other options besides filters. We can keep it, but consider making it left aligned and making the style of it `TextAppearance.OtherStuffIForgetThatStudioWillAutocomplete.Subhead`.